deploy/certmanager/060-Deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: strimzi-drain-cleaner
+  labels:
+    app: strimzi-drain-cleaner
+  namespace: strimzi-drain-cleaner
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: strimzi-drain-cleaner
+  template:
+    metadata:
+      labels:
+        app: strimzi-drain-cleaner
+    spec:
+      serviceAccountName: strimzi-drain-cleaner
+      containers:
+        - name: strimzi-drain-cleaner
+          image: quay.io/scholzj/strimzi-cistic-odpadu:latest
```

Comment:
Just as a remainder to update the image field to the official image in the Strimzi repo that will come later.

Reply:
That needs to be done when we have the CI and are actually pushing the images into the Strimzi repositories.